src/components/projects/CreateProjectForm.vue

```diff
@@ -93,6 +93,26 @@
                          v-model="targets[n-1].dec" />
             </b-field>
 
+            <b-field style="margin-left: 2em;">
+                <template #label>
+                    Hours / Degrees
+                    <b-tooltip type="is-dark" label="When this is turned on, PTR expects units in RA hours. When off, PTR expects units in RA degrees.">
+                        <b-icon size="is-small" icon="help-circle-outline"></b-icon>
+                    </b-tooltip>
+                </template>
+                <b-checkbox v-model="targets.RAhours"></b-checkbox>
+            </b-field>
+
+            <b-field style="margin-left: 2em;">
+                <template #label>
+                    Sexagesimal?
+                    <b-tooltip type="is-dark" label="When this is turned on, PTR expects units in Hours Minutes Seconds. When off, PTR expects units in Decimals.">
+                        <b-icon size="is-small" icon="help-circle-outline"></b-icon>
+                    </b-tooltip>
+                </template>
+                <b-checkbox v-model="targets.hrsminssecs"></b-checkbox>
+            </b-field>
+
```

Comment:
I think this UI element is a little unintuitive - checkboxes are definitely easiest to deal with, but I feel like a [switch](https://buefy.org/documentation/switch/) or dropdown select or some other element might work better and eliminate or reduce the need for clarifying text

Reply:
I think an elegant solution might be a text box with a select field on the right to specify the units. There's an example if you expand the advanced project settings and look at the 'RA Offset' field. That's something I'd be happy to add unless someone else wants to do it. I also think it doesn't need to block this PR. The current solution works, so we can get it shipped sooner and improve the UI as a second step. Open to alternatives if people have differing opinions.

Reply:
Yes, I like the select field on the right thing. I just didn't want to move too much around given my novice-ness at javascript. I just wanted something to work first. If there was a select field, then that text above could be deleted also which is nice. I put checkboxes in the project tab because it was easy and lots of room